bin/format_code.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+for i in `find src -type f -name '*.rs'`
+	do rustfmt --write-mode=overwrite $i
+done
```

Comment:
For some reason that doesn't seem to work on my computer. Perhaps a second commit?